R/install-dev.sh

```diff
@@ -35,12 +35,19 @@ LIB_DIR="$FWDIR/lib"
 mkdir -p $LIB_DIR
 
 pushd $FWDIR > /dev/null
+if [ ! -z "$1" ]
+  then
+    R_HOME="$1/bin"
+   else
+    R_HOME="$(dirname $(which R))"
```

Comment:
In R there there is usually no `R_HOME` like there is in python so this might confuse users.  We should use the default R when the variable is not passed. 

Or I can use a variable like `SPARK_R_HOME` which specifies the R home directory ?